src/CalcManager/Ratpack/exp.cpp

```diff
@@ -345,8 +345,14 @@ void powratNumeratorDenominator(PRAT *px, PRAT y, uint32_t radix, int32_t precis
         // ##################################
         // Round the originalResult to roundedResult
         // ##################################
+        PRAT rat_digits = nullptr;
```

Comment:
Please add a call to `destroyrat()` for `rat_digits`. Every `PRAT` which is assigned a value other than `nullptr` requires a `destroyrat()` call to avoid a memory leak.